### DISCUSSION TITLE
$CURRENT_USER as the default value on a M2O user field

### DISCUSSION THREAD

**Discussion by cory-glooh** [directus/directus#11158] [Feature Requests]
when creating a many to one relationship with a user, an option to use $CURRENT_USER as the default value, would be nice.

**Comment by benhaynes** [comment]
I agree! There is a way to set the "On Create" and On Update" value to the current user, but the **default** value is a bit different... and we should support this. Same thing for `$NOW`, etc.
![Screen Shot 2022-01-19 at 9 39 02 AM](https://user-images.githubusercontent.com/522079/150152562-74475109-8f8e-4f87-9013-c7a21e66a8e5.png)

**Reply by kevincam3** [reply]
Is there a workaround if you want to set the default value to be the current user?

**Reply by MentalGear** [reply]
Would this feature allow for n-level deep multi-tenancy within a single instance? (For example: Organisation > Team > Project)

**Comment by rijkvanzanten** [comment]
This exists as the "On Create" and "On Update" control values. There's nothing a default value allows you to do that the "On Create" doesn't achieve currently. The default value is saved to the database, which means there's no way to set it to "Current User", as the database isn't aware of what that user is! 🙂

**Reply by MarvinXu** [reply]
Maybe not as a "default value", but a "auto-fill" feature?
